index.html

```diff
@@ -348,12 +348,12 @@
                         <thead><tr><th>Device Button Changes</th><th><code>button</code></th></tr></thead>
                         <tbody>
                             <tr><td>Neither buttons nor touch/pen contact changed since last event</td><td>-1</td></tr>
-                            <tr><td>Left Mouse,<br>Touch Contact,<br>Pen contact (with no modifier buttons pressed)</td><td>0</td></tr>
+                            <tr><td>Left Mouse,<br>Touch Contact,<br>Pen contact</td><td>0</td></tr>
```

Comment:
I also think that Wacom's button swapping is a low (driver) level setting that is beyond the scope of this spec. A similar case: mouse L/R button swapping is possible is in many (all?) OSes, and quite logically UI event spec is silent about it.